docs/contributing.md

```diff
@@ -0,0 +1,53 @@
+# SSS Contributor Guideline
+
+These are recommendations when contributing to the SSS repositories.
+They consider contributions to both actual content (mostly Markdown) and source code (challenges and activities) made via Git.
+Make sure you follow them whenever you make a contribution.
+
+## First Steps
+
+Some good first steps and best practices when using Git are explained here:
+- the Git Immersion tutorial: https://gitimmersion.com/
+- the Atlassian tutorial: https://www.atlassian.com/git/tutorials/learn-git-with-bitbucket-cloud
+- this blog post on the ROSEdu Techblog: https://techblog.rosedu.org/git-good-practices.html
+
```

Comment:
Add a section on language. Say we use English: commit message, PR messages, actual content, code comments etc.